package.json

```diff
@@ -26,7 +27,7 @@
     "lint-staged": {
         "src/**/*.{ts,tsx}": [
             "eslint 'src/**/*.tsx' --fix",
```

Comment:
@gmukul01 @vitulrana  should we allow the `--fix` flag here since that modify's files under the hood and sometimes can lead to unwanted changes being pushed to the repo.

Reply:
@wsameer I don't think `--fix` flag ever changes the meaning of the code, those are always minor changes like moving it into multiline types changes.

Reply:
Yeah I have also noticed, `--fix` doesn't change major things.